lib/__tests__/remove-console-statements.js

```diff
@@ -0,0 +1,3 @@
+jest.autoMockOff();
+const defineTest = require('jscodeshift/src/testUtils').defineTest;
+defineTest(__dirname, 'remove-console-statements');
```

Comment:
Yeah I can refactor it so it's like this, but if I want to unit test the different codemod functions (like getConsoleStatments and getConsoleExpressions) then I need to split them into different files.

I'll go ahead and do that now

Reply:
So I see my confusion with this - because of the way I named it it seems like you would have to create a new file for each test, but if you use it like this:

```js
// test.js
defineTest(__dirname, 'rewriter', undefined, 'remove-console-all');
defineTest(__dirname, 'rewriter', undefined, 'remove-console-all-2');
```

Then you can test multiple fixtures for one file. Learned this while I was refactoring to allow for tests, so now you can unit test individual mods I guess 🎉